docs/index.rst

```diff
@@ -47,8 +47,14 @@ This will add a `migrations` folder to your application. The contents of this fo
 
 You can then generate an initial migration::
 
-    $ flask db migrate
-    
+    $ flask db migrate #-m "You can leave optional message."
+
+- ``Revision script without message:``
+    5d3244f628be.py
+
+- ``Revision script with message:``
+    5d3244f628be_you_can_leave_message_optional.py
```

Comment:
This is a high-level example, showing naming examples for migration scripts does not belong, and does not really add any valuable information. Better to encourage people to always use comments by only showing that in the example above, as I suggested.

Reply:
Ok!